packages/components/rate/src/rate.vue

```diff
@@ -88,10 +89,17 @@ export default defineComponent({
   setup(props, { emit }) {
     const elForm = inject(elFormKey, {} as ElFormContext)
 
+    const rateSize = useSize()
```

Comment:
`rateSize` will return `"default"` so rateSize.value will always be truthy?

Reply:
Yes, the other components have the same behavior. The buttons also have class `el-button--default`.